magicctapipe/io/tests/test_io_monly.py

```diff
@@ -17,456 +19,564 @@
 )
 
 
-def test_format_object():
-    """
-    Simple check on a string
-    """
-    str_a = "a{b[[xz,cde}"
-    str_b = format_object(str_a)
-    assert str_b == "a b  xzcde "
-
+class TestGeneral:
+    def test_format_object(self):
+        """
+        Simple check on a string
+        """
+        str_a = "a{b[[xz,cde}"
+        str_b = format_object(str_a)
+        assert str_b == "a b  xzcde "
 
-def test_save_pandas_data_in_table(temp_pandas, pd_test):
-    """
-    Check on pandas dataframe (before = after saving it)
-    """
-    out = temp_pandas / "pandas.h5"
-    save_pandas_data_in_table(pd_test, str(out), "abc", "event")
-    df1 = pd.DataFrame(np.array([[1, 2], [3, 4], [5, 6]]), columns=["a", "b"])
-    df = pd.read_hdf(str(out), key="event")
-    assert df.equals(df1)
+    def test_save_pandas_data_in_table(self, temp_pandas, pd_test):
```

Comment:
same here, also the same function is already tested in test_io.py

Reply:
removed